test/e2e/persistent_volumes.go

```diff
@@ -41,18 +42,17 @@ func completeTest(f *framework.Framework, c clientset.Interface, ns string, pv *
 	By("Checking pod has write access to PersistentVolume")
 	createWaitAndDeletePod(f, c, ns, pvc.Name)
 
-	// 3. delete the PVC, wait for PV to become "Available"
+	// 3. delete the PVC, wait for PV to become "Released"
```

Comment:
L46, L73: _By("Deleting the PVC to invoke the recycler")_. `completeTest` and `completeMultiTest` do not invoke recycling since the pv's reclaimPolicy is set to "Retain" for these test cases.